### PR TITLE
Update layout for `PriceCalculator` to match design

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Update layout in `PriceCalculator` to match final design.

Switch out usage of `InputField` (ui) to new `TextField`.

## Justify why they are needed

Complete stuff.

## Checklist before requesting a review

- [x] I have performed a self-review of my code

[GRW-1880]: https://hedvig.atlassian.net/browse/GRW-1880?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #985** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/985" target="_blank">Graphite</a> 
    * **PR #987** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/987" target="_blank">Graphite</a> 
      * **PR #989** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/989" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/989?utm_source=stack-comment).

File: apps/store/src/components/PriceCalculator/PriceCalculatorAccordion.tsx [hunk 1]
```diff
@@ -23,24 +24,30 @@ export const PriceCalculatorAccordion = ({ form, children }: Props) => {
...
+              <SpaceFlex space={0.5} align="center">
+                <StepIcon filled={section.id === activeSectionId} />
+                <StyledHeading as="h3" variant="standard.18" muted={isMuted}>
+                  {translateLabel(section.title)}
+                </StyledHeading>
+              </SpaceFlex>
+              {section.state === 'valid' && (
+                <Accordion.Trigger>
+                  <Text size="l" color="gray700">
+                    Edit
```
Comment (alebedev) [inline apps/store/src/components/PriceCalculator/PriceCalculatorAccordion.tsx:42]: Should it be localized?

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/src/components/PriceCalculator/PriceCalculatorAccordion.tsx (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/components/PriceCalculator/PriceCalculatorAccordion.tsx:42]: Ahh yes will do that later!

**Comment by robinandeer** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/989?utm_source=stack-comment).


Job Reference: JZRQe0PpwZMxwAQsDiRu

**Pull request merged**